<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="HGAJJELA02" failures="1" tests="1" name="FREE CRM Test" time="59.822" errors="0" timestamp="05 Nov 2020 13:15:45 GMT">
  <testcase classname="MyRunner.TestRunner" name="feature" time="59.459">
    <failure type="cucumber.runtime.CucumberException" message="java.lang.AssertionError: expected [CRMPRO - CRM software for customer relationship management, sales, and support.] but found [CRMPRO]">
      <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: expected [CRMPRO - CRM software for customer relationship management, sales, and support.] but found [CRMPRO]
at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
at MyRunner.TestRunner.feature(TestRunner.java:36)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
Caused by: java.lang.AssertionError: expected [CRMPRO - CRM software for customer relationship management, sales, and support.] but found [CRMPRO]
at stepDefinitions.LoginStepDefinition.user_is_on_hopme_page(LoginStepDefinition.java:62)
at âœ½.Then user is on home page(login.feature:9)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- feature -->
</testsuite> <!-- FREE CRM Test -->
